cmd/core-data/res/docker/configuration.toml

```diff
@@ -0,0 +1,55 @@
+AddToEventQueue = true
+DeviceUpdateLastConnected = false
+FormatSpecifier = '%(\\d=\\$)?([-#= 0(\\<]*)?(\\d=)?(\\.\\d=)?([tT])?([a-zA-Z%])'
+MetaDataCheck = false
+PersistData = true
+ReadMaxLimit = 100
+ServiceUpdateLastConnected = false
+ValidateCheck = false
+
+[Service]
+BootTimeout = 30000
+ClientMonitor = 15000
+CheckInterval = '10s'
+Host = 'edgex-core-data'
+Port = 48080
+Protocol = 'http'
+ReadMaxLimit = 100
```

Comment:
Why there are two ReadMaxLimit? This second one is not used in core-data, and also in core-command

Reply:
Good catch. This was an error when I merged a file conflict from another PR.